src/libopensc/card-epass2003.c

```diff
@@ -72,6 +72,9 @@ static struct sc_card_driver epass2003_drv = {
 #define KEY_LEN_DES3	24
 #define HASH_LEN	24
 
+#define SHA1_DIGEST_LENGTH 20
+#define SHA256_DIGEST_LENGTH 32
```

Comment:
`SHA_DIGEST_LENGTH` and `SHA256_DIGEST_LENGTH` are already defined in the `openssl/sha.h` included above.

Reply:
Well, then please feel free to delete lines 75--76.

I forgot to note that (while I do not have the card and I did not follow up the stack) bugs of this kind sometimes have security implications.  Perhaps it should be noted in the change log.